glean_parser/templates/markdown.jinja2

```diff
@@ -28,9 +28,27 @@ See the Glean SDK documentation for the [`{{ ping_name }}` ping]({{ ping_name|pi
 {% if ping_name|ping_send_if_empty %}
 This ping is sent if empty.
 
+{% endif %}
+{% if ping_name|ping_include_client_id %}
+This ping includes the client id.
+
+{% endif %}
+{% if ping_name|ping_data_reviews %}
+**Data reviews for this ping:**
+{% for review in ping_name|ping_data_reviews %}
+    - [{{review}}]({{review}})
+{% endfor %}
+
+{% endif %}
+{% if ping_name|ping_bugs %}
+**Bugs related to this ping:**
+{% for bug in ping_name|ping_bugs %}
+    - [Bug {{bug}}](https://bugzilla.mozilla.org/show_bug.cgi?id={{bug}})
```

Comment:
Afaik we now enforce these to be URLs regardless, as they could also point to GitHub or other places.

Reply:
Also from https://mozilla.github.io/glean_parser/pings-yaml.html:
> If a number, it is an ID to an issue in the default tracker (e.g. Mozilla’s Bugzilla instance). If a string, it must be a URI to a bug page in a tracker.

So I guess I should verify before creating the link. I am guessing this is just for backwars compatibility from when these used to be only numbers?

Reply:
Indeed, we still do allow integers, so we would need to check indeed.

Reply:
Is that only for backwards compatibility? I thought we'd made the decision to permit URLs only

Reply:
I thought so too!
I think we _did_ change all IDs to URLs in projects we can influence (citation needed) and [the glinter suggests that it's deprecated](https://github.com/mozilla/glean_parser/blob/master/glean_parser/lint.py#L142-L149).
However we didn't change the schema because we need to parse old stuff (as in: the probe-scraper parses)